vktrace/vktrace_layer/vktrace_lib_trim.cpp

```diff
@@ -4463,8 +4463,27 @@ void recreate_fences(StateTracker &stateTracker) {
 }
 
 void recreate_events(StateTracker &stateTracker) {
+    VkResult result = VK_EVENT_RESET;
+    vktrace_trace_packet_header *pheader = nullptr;
     for (auto obj = stateTracker.createdEvents.begin(); obj != stateTracker.createdEvents.end(); obj++) {
         vktrace_write_trace_packet(obj->second.ObjectInfo.Event.pCreatePacket, vktrace_trace_get_trace_file());
+        result = mdd(reinterpret_cast<void *>(obj->second.belongsToDevice))
+                     ->devTable.GetEventStatus(obj->second.belongsToDevice, reinterpret_cast<VkEvent>(obj->second.vkObject));
+        switch (result) {
+            case VK_EVENT_RESET:
+                pheader =
+                    generate::vkResetEvent(false, obj->second.belongsToDevice, reinterpret_cast<VkEvent>(obj->second.vkObject));
```

Comment:
When the event is created, its initial state will be unsignaled, so it shouldn't be necessary to call vkResetEvent after vkCreateEvent.